### PR TITLE
[PAY-1569] Update blog post link

### PR DESCRIPTION
**Pull request by piazzatron**
### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

Testing locally

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/piazz-link-to-blog

File: packages/web/src/components/inbox-unavailable-modal/InboxUnavailableModal.tsx [hunk 1]
```diff
@@ -110,7 +111,8 @@ export const InboxUnavailableModal = ({
       dispatch(createChat({ userIds: [user.user_id] }))
       onClose()
     } else {
-      // TODO: Link to blog post
+      window.open(CHAT_BLOG_POST_URL, '__blank')
```
Comment (rickyrombo) [inline packages/web/src/components/inbox-unavailable-modal/InboxUnavailableModal.tsx:114]: `_blank`?

**Review by rickyrombo — APPROVED** [review]
I think you have two underscores in `_blank`?

File: packages/web/src/components/inbox-unavailable-modal/InboxUnavailableModal.tsx (hunk 1 above)
Comment (piazzatron) [reply packages/web/src/components/inbox-unavailable-modal/InboxUnavailableModal.tsx:114]: indeed, fixed in another place as well. copy and paste error

**Pull request merged**